packages/expo-modules-autolinking/scripts/ios/xcode_env_generator.rb

```diff
@@ -16,6 +16,7 @@ def maybe_generate_xcode_env_file!()
     Pod::UI.warn "Unable to generate `.xcode.env.local` for Node.js binary path: #{stderr}"
   else
     Pod::UI.info "Auto-generating `.xcode.env.local` with $NODE_BINARY=#{node_path}"
-    File.write(xcode_env_file, "export NODE_BINARY=\"#{node_path}\"\n")
+    File.write(xcode_env_file, "export NODE_BINARY=\"#{node_path}\"\nif [[ \"$EX_UPDATES_NATIVE_DEBUG\" = \"1\" ]]; then\n  export FORCE_BUNDLING=1\n  unset SKIP_BUNDLING\n  export RCT_NO_LAUNCH_PACKAGER=1\nfi\n")
```

Comment:
the generated content will be something like:
```
export NODE_BINARY="/path/to/node"
if [[ "$EX_UPDATES_NATIVE_DEBUG" = "1" ]]; then
 export FORCE_BUNDLING=1
 unset SKIP_BUNDLING
 export RCT_NO_LAUNCH_PACKAGER=1
fi
```
no matter the current `$EX_UPDATES_NATIVE_DEBUG` is defined or not. so that the `$EX_UPDATES_NATIVE_DEBUG` will be resolved from xcode build time right?

please note that xcode start from macos finder may not read environment variables from shell rc files as expected.

maybe we should just check `$EX_UPDATES_NATIVE_DEBUG` when running `pod install` time, i.e.:
  - when `EX_UPDATES_NATIVE_DEBUG=0 pod install`, the generated content is only `export NODE_BINARY=...`
  - when `EX_UPDATES_NATIVE_DEBUG=1 pod install`, the generated content will be `export NODE_BINARY=...\nexport FORCE_BUNDLING=1...`.

then it will come with another side effect. if people want to change `EX_UPDATES_NATIVE_DEBUG=0`, they must run `pod install` again. people may not aware that they leave the debugging settings inside the `.xcode.env.local`. even more, these settings will leave there if people remove the expo-updates package.
if you go with this approach, please also add more information log that we are overriding these react-native environment variables. as soon as finishing debugging, they should run `pod install` again.

Reply:
The problem with your suggestion is that this script does not rewrite `.xcode.env.local` if it already exists. So if I unset `EX_UPDATES_NATIVE_DEBUG` and run `pod install` again, the `.xcode.env.local` is not updated to remove the additional text.

I'll give this some more thought, to see if there is a better way of accomplishing this. Maybe the updates config plugin needs its own `.xcode.env.updates` file.

Reply:
> The problem with your suggestion is that this script does not rewrite .xcode.env.local if it already exists. So if I unset EX_UPDATES_NATIVE_DEBUG and run pod install again, the .xcode.env.local is not updated to remove the additional text.

we can change that behavior, e.g. comparing .xcode.env.local contents with current EX_UPDATES_NATIVE_DEBUG and update .xcode.env.local if necessary.

Reply:
That could work, but I think users will expect a file that ends in `local` to be customizable, so that we should not change it again once it exists. Probably safer to keep the updates changes in a separate file.